### DISCUSSION TITLE
Add RadioBrowser to the radios

### DISCUSSION THREAD

**Discussion by d-faure** [strawberrymusicplayer/strawberry#1065] [Ideas]
https://www.radio-browser.info/
> This is a community driven effort (like wikipedia) with the aim of collecting as many internet radio and TV stations as possible. Any help is appreciated!
> 
> * Free for ALL!
> * Data license: public domain, software license: GPL, server software: open source
> * Open API for usage in own apps
> * Everyone is free to use the collected data (station names, tags, links to stream, links to homepages, language, country, state) in their works. I give all the rights I have at the accumulated data to the public domain.
> 

And it has an API: https://api.radio-browser.info/

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
